[RHDHBUGS-2768]: Document dynamic plugin mirroring for air-gapped environments (#2262)
redhat-developer/red-hat-developers-documentation-rhdh: modules/install_installing-rhdh-in-an-air-gapped-environment/snip-mirror-dynamic-plugins-partially-disconnected.adoc

:_mod-docs-content-type: SNIPPET
. Download the plugin mirroring script by running the following command:
+
[source,terminal,subs="+attributes,+quotes"]
----
$ curl -sSLO https://raw.githubusercontent.com/redhat-developer/rhdh-operator/refs/heads/release-{product-version}/.rhdh/scripts/mirror-plugins.sh
----

. Mirror the plugin catalog index and all referenced plugin OCI artifacts to your target registry by running the following command:
+
[source,terminal,subs="+attributes,+quotes"]
----
$ bash mirror-plugins.sh \

>   --plugin-index oci://registry.redhat.io/rhdh/plugin-catalog-index:{product-version} \

Reviewer: ```suggestion
  --plugin-index oci://registry.access.redhat.com/rhdh/plugin-catalog-index:{product-version} \
```

  --to-registry _<target_registry>_
----
+
where:
+

> `_<target_registry>_`:: Enter the URL of the target mirror registry, for example, `registry.example.com`.

Reviewer 2: ```suggestion
`_<target_registry>_`:: Enter the URL of the target mirror registry, such as, `registry.example.com`.
```
(For consistency with the rest of the doc)
